### PR TITLE
Mvbe 11035/extend mail address field length

### PR DESCRIPTION
**Pull request by mhemeryck**
Link: https://vikingco.atlassian.net/browse/MVBE-11035

Extends the mail `to_address` and `from_address` from length 50 to 255.
Additionally, some boilerplate code was added.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```
Comment (Jogazo) [inline manage.py]: @mhemeryck  Can you tell me on why you need this file?

Comment (mhemeryck) [inline manage.py]: Used this to generate the migrations automatically. Basically got it from the original repo this one was forked from.

Comment (Jogazo) [inline manage.py]: OK, I was just curious :)

Comment (mhemeryck) [inline manage.py]: Apart from the management script, there are some other changes that would be required to make this repo self-containing (deployable), e.g. a requirements file and the setup script linking to this. Didn't add this though, as it is out of the scope of this ticket.

For future reference: I did ran the management script with a clean virtualenv, only having `Django==1.9.7` in it.

**Pull request merged**